### PR TITLE
Add avrio to USERS.md

### PR DESCRIPTION
**Pull request by ArielH25**
Add Avrio's use case of RocksDB to USERS.md

### CODE REVIEW

File: USERS.md [hunk 1]
```diff
@@ -96,3 +96,6 @@ LzLabs is using RocksDB as a storage engine in their multi-database distributed
 
 ## IOTA Foundation
  [IOTA Foundation](https://www.iota.org/) is using RocksDB in the [IOTA Reference Implementation (IRI)](https://github.com/iotaledger/iri) to store the local state of the Tangle. The Tangle is the first open-source distributed ledger powering the future of the Internet of Things.
+ 
+ ## Avrio Project
+ [Avrio Project](http://avrio-project.github.io/avrio.network/) is using RocksDB in [Avrio ](https://github.com/avrio-project/avrio) to storeBlocks, account balances and data and other blockchain releated data. Avrio is a multiblockchain decentrilsed cryptocurrency empowering monitary transactions.
```
[the remark this answers is not in the document]
Comment (ArielH25) [reply USERS.md]: sorry, it should be:
store blocks, blockchain-related data, decentralized and  monetary

[the remark this answers is not in the document]
Comment (ArielH25) [reply USERS.md]: I added a new commit fixing these issues

**Comment by ArielH25** [comment]
sure hang on

**Comment by ArielH25** [comment]
done @riversand963

File: USERS.md [hunk 2]
```diff
@@ -96,6 +96,8 @@ LzLabs is using RocksDB as a storage engine in their multi-database distributed
 
 ## IOTA Foundation
  [IOTA Foundation](https://www.iota.org/) is using RocksDB in the [IOTA Reference Implementation (IRI)](https://github.com/iotaledger/iri) to store the local state of the Tangle. The Tangle is the first open-source distributed ledger powering the future of the Internet of Things.
- 
+## Avrio Project
+ [Avrio Project](http://avrio-project.github.io/avrio.network/) is using RocksDB in [Avrio ](https://github.com/avrio-project/avrio) to store blocks, account balances and data and other blockchain-releated data. Avrio is a multiblockchain decentralized cryptocurrency empowering monetary transactions.
 ## Crux
 [Crux](https://github.com/juxt/crux) is a document database that uses RocksDB for local [EAV](https://en.wikipedia.org/wiki/Entity%E2%80%93attribute%E2%80%93value_model) index storage to enable point-in-time bitemporal Datalog queries. The "unbundled" architecture uses Kafka to provide horizontal scalability.
+
```
Comment (riversand963) [inline USERS.md]: Remove this line?

File: USERS.md [hunk 3]
```diff
@@ -96,6 +96,8 @@ LzLabs is using RocksDB as a storage engine in their multi-database distributed
 
 ## IOTA Foundation
  [IOTA Foundation](https://www.iota.org/) is using RocksDB in the [IOTA Reference Implementation (IRI)](https://github.com/iotaledger/iri) to store the local state of the Tangle. The Tangle is the first open-source distributed ledger powering the future of the Internet of Things.
- 
+## Avrio Project
```
Comment (riversand963) [inline USERS.md]: Empty line between each user

**Comment by ArielH25** [comment]
is there anything else to do?

**Comment by riversand963** [comment]
Can't think of anything significant.

**Comment by ArielH25** [comment]
@riversand963 yes

**Comment by riversand963** [comment]
Cool. Could you address the review comments, please? After that, we can merge if there is no other issue.

**Comment by ArielH25** [comment]
ok hang on

**Comment by ArielH25** [comment]
done

**Pull request closed without merging**